src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -53,21 +56,21 @@ class SessionService(
     }.flatten()
```

Comment:
Naming is wrong they might not be available

` var available = sessionTemplates.map {`

I would refactor all this (line 51 to 61) into a method called getAvailableSessions() not sure how it works with null prisonerId need a call

Reply:
Was originally available sessions for the period from the session templates, not available to book - just what sessions are in that time period, with the filtering now has a double meaning.  Ill update the name to remove any implied meaning.